[mh-68] bugfix

sharrisoncaktus opened the pull request:
Not much to say: A simple javascript typo.

sharrisoncaktus committed:
[mh-68] bugfix

The event is "e" not "event". Also reference the form by id.

dsummersl commented:
Can one of the admins verify this patch?

nmashton commented on myhpom/templates/myhpom/accounts/choose_network.html:
```diff
@@ -139,11 +139,12 @@ <h4>
         $customProvider.on('keypress', function (e) {
             $fullSelect.val('null');
             $radios.prop('checked', false);
+            $('#id_healthcare_network').val('');
```
Functionally it doesn't make much difference, but it'd be Nice to move these up in the scope so that you just have one `var $healthcareNetworkInput = $('#id_healthcare_network');` statement (same goes for the next change below) rather than hitting the DOM afresh each time you press a key.

sharrisoncaktus replied:
That makes sense. I'll do that and merge it.

nmashton reviewed (approved):
It looks like it works! I have a comment that doesn't really affect the functioning of it. :shipit:

sharrisoncaktus committed:
[mh-68] store healthcare_network element in a variable

avoid hitting the DOM on every keypress in custom_provider, and then we might as well use it everywhere.

sharrisoncaktus commented:
(00:16:27 later...)
